SmartCharger.Data/Entities/Charger.cs

```diff
@@ -0,0 +1,20 @@
+﻿
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace SmartCharger.Data.Entities
+{
+    public class Charger : BaseEntity
+    {
+        public string Name { get; set; }
+        public double Latitude { get; set; }
+        public double Longitude { get; set; }
+        public  DateTime Created { get; set; }
```

Comment:
I suggest a different name for this variable so the name specifies that it is referring to time when it was created. (for example, CreationTime)

Reply:
I agree, thank you.  In new commit I will change the name of the variable.